sites/bioopticsworld/server/routes/index.js

```diff
@@ -17,6 +18,10 @@ module.exports = (app) => {
   // Content Types
   contentTypes(app);
 
+  // Search
+  app.get('/search', (_, res) => { res.marko(search); });
```

Comment:
These should move above content and page routes.
What the purpose of the `/search/*` route?